src/Humanizer.Tests/TimeSpanHumanizeTests.cs

```diff
@@ -82,8 +82,8 @@ public void TimeSpanWithMaxTimeUnit(int ms, string expected, TimeUnit maxUnit)
         }
 
         [Theory]
-        [InlineData(0, 3, "no time")]
```

Comment:
Any reasons why you have changed "no time" to "just now"? I'm asking because this can be used with general time-spans, which are not bound to any particular momentum.